Project/Source Files/Seach/SearchProjectRequest.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  SearchRequest.swift
+//  NetguruCollegeApp
+//
+
+import Foundation
+
+final class SearchProjectRequest: APIRequest {
+    var path: String = "/search/repositories"
+    var method: APIMethod = .get
+    var parameters: [String: String] = [:]
```

Comment:
how about creating initializer where we will take parameters dictionary as parameter?
That way you will be able to initialize `SearchProjectRequest` in one line and later no one will forget to set parameters when using this request